src/datastation/open_access_archeodepot.py

```diff
@@ -125,10 +123,28 @@ def all_files_found_in_list(files_to_check, all_files):
     def get_filepaths(files):
         return list(map(lambda file: file_path(file), files))
 
-    def change_file(restricted_value: bool, files):
+    def file_to_file_meta_update(file, restricted_value):
+        update = {
+            "id": file['dataFile']['id'],
+            "label": file['label'],
+            "directoryLabel": file.get('directoryLabel', ""),
+            "description": file.get('description', ""),
+            "categories": file.get('categories', []),
+            "provFreeForm": ""
+        }
+        # Setting restrict to the same value causes an error in the API
+        if file.get('restricted') != restricted_value:
```

Comment:
Tried it and it appears the function is only called for files that need to be changed. So the function can be reduced to

    return {"id": file['dataFile']['id'], "restrict": restricted_value}

Reply:
Without the dry run I get 

    "status":"ERROR","code":405,"message":"API endpoint does not support this method.